provider/testdata/data/postcard/fingerprints.csv

```diff
@@ -887,7 +887,7 @@ props/Join_C@1, und, 10B, ca93d1f94fc9d2af
 props/LOE@1, und, 58B, dbe045415d3b363d
 props/Lower@1, und, 5371B, 7c979c7a7818dd83
 props/Math@1, und, 1107B, ebcaa9adf44f53ad
-props/NChar@1, und, 139B, fc1f0bde52352192
+props/NChar@1, und, 147B, 9e8b82fc274be65
```

Comment:
note: the data changes are happening because  these properties had trie segments that went up to `char::MAX`, which were not being handled before